INSTALL.rst

```diff
@@ -52,11 +52,49 @@ results, you will additionally need:
 * The Slater-Koster data used in the tests (see below)
 
 
+Successful builds
+-----------------
+
+DFTB+ is regularly built and tested in following build environments:
```

Comment:
Table isn't clear that compilation without MPI is also possible. Suggest something like

DFTB+ is regularly built and tested for both serial and MPI  environments on the following architectures: